examples/example-todo-app/src/app/page.tsx

```diff
@@ -1,101 +1,15 @@
-import Image from "next/image";
+'use client';
```

Comment:
The page in NextJS must not be a client component. Find a way to refactor to server component